src/datasources/network/entities/network.request.entity.ts

```diff
@@ -1,4 +1,6 @@
+type Primitive = string | number | bigint | boolean | undefined | symbol | null;
```

Comment:
Nit: This type seems quite general. I can’t see other use cases for it right now, but what do you think about moving it to a separate file for potential reuse as a general type?

Reply:
I oversaw this before merging. For future reference: we subsequently discussed this and decided to leave it here for now. We will move it if we later need the type.